monkey/infection_monkey/network_scanning/scan_target_generator.py

```diff
@@ -58,14 +58,17 @@ def _range_to_addresses(range_obj: NetworkRange) -> List[NetworkAddress]:
     return addresses
 
 
-def _get_ips_from_ranges_to_scan(ranges_to_scan: List[str]) -> List[NetworkAddress]:
-    scan_targets = []
-
+def _get_ips_from_subnets_to_scan(subnets_to_scan: List[str]) -> List[NetworkAddress]:
     ranges_to_scan = NetworkRange.filter_invalid_ranges(
-        ranges_to_scan, "Bad network range input for targets to scan:"
+        subnets_to_scan, "Bad network range input for targets to scan:"
     )
 
     network_ranges = [NetworkRange.get_range_obj(_range) for _range in ranges_to_scan]
```

Comment:
`get_range_obj` is where a lot of things could be simplified. It uses a bunch of hand-made classes, like `IpRange` or `CidrRange`. In reality, those could probably be replaced by `IPv4Network` objects. But one PR at a time, let's not bloat this one

Reply:
+1

Reply:
+1